coordinator/tests/resources/test-data/practitioner-role/task.ts

```diff
@@ -0,0 +1,40 @@
+import {fhir} from "@models"
+
+export const practitionerRoleTask: fhir.PractitionerRole = {
```

Comment:
can we re use other practitionerRoles?

Reply:
Potentially with some value manipulation, but we again run into the issue of the values differing across each message type

Reply:
wonder if its worth syncing up the values that are inconsistent between example messages. people are probs gonna read this and be confused why we have different test data